.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+name: Test and Deploy
+on: push
```

Comment:
I found these by looking at https://github.com/search?l=YAML&p=2&q=org%3Awmde+release&type=Code

Reply:
it uses [this action](https://github.com/JS-DevTools/npm-publish) which by default only deploys on changes to `package.json`. I tried to make it clearer by using the explicit parameter `check-version` (although it's true by default) but it might make sense to document it in a clearer way somehow

Reply:
or maybe leave it out altogether and use what other repos use for clarity's sake?

Reply:
Thank you for taking care of this :pray: